gitgud/__main__.py

```diff
@@ -68,13 +69,13 @@ def __init__(self):
         ])
         
         show_description = "".join([
```

Comment:
Is there a reason we're not already just doing `"\n.join...`? Seems like it might be a better option

Reply:
The reason that's there is because we've got more formatting in each individual line coming up for each subcommand. I was unsure about cramming together the escape characters with the actual content in the same line. If we switch to "`\n".join(...)`, we'd go from:
`"    tree", "\t", "Show the current state of the branching tree"`
to 
`"    tree\tShow the current state of the branching tree"`

If you'd like, @benthayer , I can switch it anyway.

Reply:
Yeah, that's how it's normally how we do it. I think it's better than having so many newlines